### DISCUSSION TITLE
小麥注音 2.5 版

### DISCUSSION THREAD

**Discussion by lukhnos** [openvanilla/McBopomofo#360] [Announcements]
- 新增 Big5 內碼輸入，按 Ctrl-` 就能輸入內碼 (@zonble, #355)
- 詞庫更新與讀音修正 (@mjhsieh, #353)
- 手動檢查版本後，若不需要更新，顯示對話盒說明原因 (#358)
- macOS 版本最低需求升至 10.13 版 (#344)

安裝包 MD5 hash: `670639980a6566f13de52f0fdbb86318`

<hr /><em>This discussion was created from the release <a href='https://github.com/openvanilla/McBopomofo/releases/tag/2.5'>2.5</a>.</em>

**Comment by Dempsey99** [comment]
請問一下，我記得以前有『取消』空白鍵選字的功能
怎麼後來的2.5找不到？

**Reply by tianjianjiang** [reply]
@Dempsey99 是指如下圖所示的選項嗎？
<img width="475" alt="Screenshot 2023-12-04 at 15 12 05" src="https://github.com/openvanilla/McBopomofo/assets/4812544/823a1b1d-271e-4830-96cb-939d7b9e5b07">

**Comment by Dempsey99** [comment]
不好意思～我眼瞎沒看到.....抱歉
確實有這個選項。

**Reply by tianjianjiang** [reply]
@Dempsey99 不會不會，別在意。 :)
